test/e2etest/bitbucket_test.go

```diff
@@ -41,7 +41,7 @@ func assertBitbucketNfsConnectivity(t *testing.T, testConfig TestConfig) {
 	returnCode, kubectlError := k8s.RunKubectlAndGetOutputE(t, kubectlOptions,
 		"exec", "bitbucket-nfs-server-0",
 		"--", "/bin/bash",
-		"-c", "echo \"Greetings from an NFS\" >> $(find /srv/nfs/bitbucket-* | head -1)/nfs-file-share-test.txt; echo $?")
+		"-c", "echo \"Greetings from an NFS\" >> $(find /srv/nfs/ | head -1)/nfs-file-share-test.txt; echo $?")
```

Comment:
We don't use the `subpath` for shared home anymore, therefore we are expecting the shared home content to be in the root of the nfs shared home volume.

Reply:
This can be okay based on the assumption that re-installing the nfs will move the data from last installation. I personally don't expect the data is persistent after nfs reinstall, but it worth to test?

Reply:
The shared home and its volume are destroyed. I have tested that. Local home still hangs around, but that was the case before as well.